### ISSUE TITLE
User with separate accounts in Magento and woocommerce is considered migrated if he starts the flow

### ISSUE THREAD

**Issue by encius** [expressly/magento#18]
When user accepts offer, new entry added in database with status = MIGRATED and even if customer is not technically migrated, product is added in the cart and coupon applied to totals. User is then able to purchase the product with discount.

**Comment by mgsmith57** [comment]
I don't believe this is an issue as the migration is an indication that the user has been migrated to the new shop, not that they have purchased. 

You will find that the user exists in the shop after they press Accept Offer / OK even if they do not complete the purchase.

Please confirm that the user exists in the shop database when the status is migrated. If so you can close the issue otherwise you may leave it often.

**Comment by encius** [comment]
user exists in both shops, but not because he was migrated from shopA to shopB. is it ok that a new entry with status MIGRATED is added in the DB and that he gets the coupon applied?

**Comment by mgsmith57** [comment]
Interesting. For the status to be migrated it would require that the receiving shop created a user in the receiving db.

Can you please add the email address and campaign customer uuid in question to the ticket so we can investigate when we analyze your test appliance.

Also please add specific steps to reproduce in all bug reports.
